layouts/shortcodes/contributors.html

```diff
@@ -0,0 +1,32 @@
+{{ $urlPre := "https://api.github.com/repos" }}
+{{ $username := "Apache" }}
+{{ $repo := "camel-website"}}
+{{ $token := ""}}
+<!-- Enter token here -->
+{{ $scope := "all"}}
+{{ if eq $token "" }}
+<p>Unable to get contributors details[set token]</p>
+{{else}}
+{{ $collaborators := getJSON $urlPre "/" $username "/" $repo "/collaborators?access_token=" $token  }}
+
+<table class="tableblock frame-all grid-all stretch">
+    <thead>
+        <tr>
+            <th>Name</th>
+            <th>Organisation</th>
```

Comment:
I'd use non British spelling, seems to be more commonly used

```suggestion
            <th>Organization</th>
```